modules/edge-machine-pools-aws-local-zones.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: CONCEPT
+[id="edge-machine-pools-aws-local-zones_{context}"]
+= Edge compute pools and AWS Local Zones
+
+Edge worker nodes are tainted worker nodes that run in AWS Local Zones locations.
+
+When deploying a cluster that uses Local Zones:
+
+* Amazon EC2 instances in the Local Zones are more expensive than Amazon EC2 instances in the Availability Zones
```

Comment:
I can't find a specific reference for this, but generally when you have a bulleted list you want consistent ending punctuation. The other bullet items in this list have a terminal period, so I'd add one to the first bullet point for consistency.

Reply:
ISG has this, I believe.